docs/src/starknet/account.md

```diff
@@ -9,7 +9,7 @@ and the second one deploys it to the network. After deployment, account can be u
 To remove an account from the accounts file, you can use  `sncast account delete`. Please note this only removes the account information stored locally - this will not remove the account from Starknet.
 
 > 💡 **Info**
-> Currently, only OpenZeppelin account creation is supported.
+> Currently, only OpenZeppelin account creation and deployment is supported.
```

Comment:
Though we list both oz and argent in the cli, no?

Reply:
But only for the `account add` command at the moment.

Reply:
I would make it a table instead with collumns:
Full name, cli arg, supported commands.

Reply:
It would only make things more confusing in my opinion. The current sentence seems straightforward, but maybe it could be rephrased a bit e.g. `Currently, creation and deployment are only supported for OpenZeppelin accounts.`

> This sentence suggests bravos is not supported, it may be misleading.

Creation and deployment for braavos accounts are not supported at the moment.